data/scenarios/Tutorials/stock.yaml

```diff
@@ -63,10 +63,10 @@ robots:
       - [100, rock]
 world:
   palette:
-    'Ω': [grass, null, base]
-    '.': [grass]
-    '_': [stone]
-    '+': [stone, wall]
+    "Ω": [grass, null, base]
+    ".": [grass]
+    "_": [stone]
+    "+": [stone, wall]
```

Comment:
My editor does this automatically so it's a little annoying!

Yaml spec says single or double quotes is fine there is just some differences to how quoting is handled. 

Happy to change back if this is too unrelated.